scenes.go

```diff
@@ -389,21 +440,26 @@ func scene10() (*config, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &config{
-		Start:      startInput,
-		Goal:       spatialmath.NewPose(goalPt, startPose.Orientation()),
-		RobotFrame: model,
-		WorldState: worldState,
+
+	return &sceneConfig{
+		Start:       startInput,
+		Goal:        spatialmath.NewPose(goalPt, startPose.Orientation()),
+		FrameToPlan: "arm",
```

Comment:
[nit] might be easier just to make this the frame itself?

Reply:
Just going to leave this one as for the moment, I toyed around with that and it wasn't as nice to read.